product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,249 @@
+
+### Install the packages
+
+The first step is to install the packages. For each Postgres package, there is a `edb-bdr5-<postgresversion>` package to go with it. 
+In our example, we are installing EDB Postgres Advanced Server (epas) version 16, and so will install `edb-bdr5-epas16`. 
+
+There are two other packages to also install: `edb-pgd5-proxy` for PGD Proxy and `edb-pgd5-cli` for the PGD command line tool.
+
+To install all of these packages on a RHEL or RHEL compatible Linux, run:
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+```
+
+### Initializing and starting the database
+
+Now initialise the database. You will need to know the path to the setup script for your particular Postgres. 
+For EDB Postgres Advanced Server, this script can be found in `/usr/edb/as16/bin` as `edb-as-16-setup`. 
+This command needs to be run with the `initdb` parameter and we need to pass an option setting the database to use UTF-8.
+
+
+```
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+```
+
+Once the database is initialised, we will start it which will enable us to continue configuring the BDR extension.
+
+```
+sudo systemctl start edb-as-16
+```
+
+### Configuring the BDR extension
+
+Installing EDB Postgres Advanced Server creates a system user `enterprisedb` with admin capabilities when connected to the database.
+In the following steps, we use the sudo command to take on the role of this user and then run the psql command to connect to the default `edb` database:
+
+```
+sudo -u enterprisedb psql edb -c <SQL COMMAND>
+```
+
+#### Preload the BDR library
+
+We want the bdr library to be preloaded with other libraries. 
+EPAS has a number of libraries already preloaded, so we have to prefix the existing list the the BDR library.
+
+```
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET shared_preload_libraries TO '\$libdir/bdr','\$libdir/dbms_pipe','\$libdir/edb_gen','\$libdir/dbms_aq'"
+```
+
+The ALTER SYSTEM SET command adds the configuration change to the `postgresql.auto.conf` file.
+
+#### Set the `wal_level`
+
+The BDR extension needs to set the server to perform logical replication. We do this by settin `wal_level` to `logical`.
+
+```
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET wal_level TO 'logical'"
+```
+
+#### Enable commit timestamp tracking
+
+The BDR extension also needs the commit timestamp tracking enabled.
+
+```
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET track_commit_timestamp TO 'on'"
+```
+
+#### Add a password to the Postgres enterprisedb user
+
+To allow connections between nodes, a password needs to be set on the Postgres enterprisedb user. 
+For this example, we are using the password `secret`.
+Select a different password for your deployments.
+You will need this password when we get to [Connecting the Cluster](05-connecting-cluster).
+
+```
+sudo -u enterprisedb psql edb -c "ALTER USER enterprisedb WITH PASSWORD 'secret'"
```

Comment:
This is one way of authentication; everyone should use their preferred authentication method. We are only showing the simplest thing.

Reply:
This is meant to be a "reasonably quick start`... we can't show every authentication path.

Reply:
Ok, so if this is a quick start then all is good.